.github/scripts/commit_and_amend.sh

```diff
@@ -0,0 +1,22 @@
+readonly GIT_HAS_CHANGES=1 # 0 = clean, 1 = dirty
```

Comment:
Since this script is also pushing to the repo, should we rename this to `amend_and_push`?